### PR TITLE
fix(rpipico): fixed declaration of setup and loop when not using leds

### PR DESCRIPTION
**Pull request by cazeaux**
This is to fix the issue #18

### CODE REVIEW

File: DahlDesignDDC/20_Setup.ino [hunk 1]
```diff
@@ -115,5 +115,10 @@ void setup()
 {
   setupLeds();
   LEDStartup();
+}
+#elif
```
Comment (andreasdahl1987) [inline DahlDesignDDC/20_Setup.ino:119]: Might need to add `(BOARDTYPE == 2)` on this `#elif`, or we might have issues with other boards?

File: DahlDesignDDC/30_Switches.ino [hunk 2]
```diff
@@ -27,4 +27,9 @@
   {
     processCommands();
   }
+#elif
```
Comment (andreasdahl1987) [inline DahlDesignDDC/30_Switches.ino:30]: Might need to add `(BOARDTYPE == 2)` on this `#elif`, or we might have issues with other boards?

**Review by andreasdahl1987 — COMMENTED** [review]
Had a look

File: DahlDesignDDC/30_Switches.ino (hunk 2 above)
Comment (cazeaux) [reply DahlDesignDDC/30_Switches.ino:30]: Damn you’re right, I was too quick on that one. I fix that rapidly.

**Pull request merged**